controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -473,6 +478,15 @@ func getFailureDomainKeyFromStorageClassParameter(sc *storagev1.StorageClass) st
 	}
 }
 
+func (r *OCSInitializationReconciler) allowRemoteStorageConsumers() bool {
+	for _, sc := range r.clusters.GetInternalStorageClusters() {
```

Comment:
This creates an in memory copy of each storage cluster returned in the slide from `GetInternalStorageClusters` 
Please do
```suggestion
	clusters := r.clusters.GetInternalStorageClusters()
	for i := range clusters {
		sc := &clusters[i]
```

Reply:
done.